docker-compose.yaml

```diff
@@ -10,28 +10,68 @@ services:
         build:
             context: .
             dockerfile: ./Dockerfile
-        command: sh -c "python manage.py runserver"
-        # volumes:
-        #     - .:/univast
+        command: sh -c "python manage.py migrate --noinput && celery -A Univast worker -l info & gunicorn --bind=0.0.0.0:9000 --timeout 600 Univast.wsgi --reload"
```

Comment:
There's no need to include the start celery worker command anymore. It has it's own service.